rules/rules.js

```diff
@@ -330,6 +332,7 @@ const getTriggeredData = function (input) {
             d.receivedTrigger = evArr[2];
             break;
         default:
+            d.module = "";
```

Comment:
why are you setting this here? Isn't this already defined above ?

Reply:
for cron triggers input.get("module") is undefined, so setting it here too

Reply:
i'm guessing you mean it will be null, not undefined?  Is that a problem?  I think null would be better then an empty string ?

Reply:
True, let's get rid of this change